src/components/container/sticky-bar/sticky-bar.scss

```diff
@@ -0,0 +1,56 @@
+@use '../../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  display: contents;
+}
+
+.sbb-sticky-bar {
+  position: sticky;
+  display: flex;
+  width: 100%;
+  bottom: 0;
+  background-color: var(--sbb-sticky-bar-bg-color);
+  padding-block: var(--sbb-spacing-responsive-xxxs);
```

Comment:
I asked Florian if we really should provide a default padding. I remember we once decided to not include any default padding.